meshery

```diff
@@ -162,7 +178,13 @@ else
 fi
 
 
-printf "Removing installation files and opening Meshery..."
+printf "Removing installation files"
 rm -rf meshery.zip ${PWD}/$temporary_dir_name/
 
-mesheryctl system start --yes -p $PLATFORM
+if [ "$DEPLOY_MESHERY" == false ] ;
+then
+    printf "\nmesheryctl command installation is complete.\n"
```

Comment:
We say mesheryctl is installed in line 146. Instead we could say "Run mesheryctl system start" to start Meshery. Or something along those lines.